xds/src/test/java/io/grpc/xds/WeightedTargetLoadBalancerTest.java

```diff
@@ -179,6 +179,7 @@ public void tearDown() {
     }
   }
 
+  @Deprecated
```

Comment:
We want to fix these. I've sent out https://github.com/grpc/grpc-java/pull/12053